### PR TITLE
Add an option to hide when leaving hover

### LINKED ISSUE #51: Add an option to leave the tooltip open on mouseout
Currently, with `showOnHover`, the tooltip gets hidden when the mouse is out of the target element.

It would be good to have an option to leave it open (and use the close button when needed), particularly if there is content to be interacted with in the tooltip.

### PR DESCRIPTION
**Pull request by chee**
closes #51  

here's an option for it, we can decide if we want close this issue by merging it or deciding not to merge it.

### CODE REVIEW

File: test/Tooltip.test.js [hunk 1]
```diff
@@ -134,6 +134,19 @@ describe("Tooltip", () => {
 				proclaim.isTrue(showStub.called);
 				proclaim.isTrue(closeStub.called);
 			});
+
```
Comment (chee) [inline test/Tooltip.test.js:137]:
```suggestion

```

**Comment by notlee** [comment]
Looks like with `showOnHover` set  `o-tooltip` fails to meet [WCAG 2.1 requirements for hover/focus](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus). 

>The intent of this condition is to ensure that additional content which may appear on hover of a target may also be hovered itself. Content which appears on hover can be difficult or impossible to perceive if a user is required to keep their mouse pointer over the trigger. When the added content is large, magnified views may mean that the user needs to scroll or pan to completely view it, which is impossible unless the user is able to move their pointer off the trigger without the additional content disappearing. 
- https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

It sounds like Lilly had the same aim "particularly if there is content to be interacted with in the tooltip".

Instead of adding a new option, perhaps we could meet both needs by making it so hovering the tooltip itself, as well as the target, keeps the tooltip open?

**Comment by notlee** [comment]
fyi we have an [issue to review o-tooltip accessibility](https://github.com/Financial-Times/o-tooltip/issues/75).

**Comment by chee** [comment]
ok

**Pull request closed without merging**